doc/release-notes/iceoryx2-unreleased.md

```diff
@@ -275,3 +275,8 @@
 15. Renamed `AttributeVerifier::keys()` to `AttributeVerifier::required_keys()`
 
 16. Renamed `AttributeVerifier::attributes()` to `AttributeVerifier::required_attributes()`
+
+17. Attribute keys and values are now `SemanticStrings`
```

Comment:
Can you add a code example please with old and new like in the other cases.

Reply:
@elfenpiff Added an example for one of the APIs, but the change is similar (the same?) for all other instances.